### PR TITLE
FSET-802 Fix parsing of guaranteedInterview flag in OnlineRepositorySpec.

### PR DESCRIPTION
**Pull request by benjstephenson**

### CODE REVIEW

File: it/repositories/OnlineTestRepositorySpec.scala [hunk 1]
```diff
@@ -99,7 +99,7 @@ class OnlineTestRepositorySpec extends MongoRepositorySpec {
   }
 
   "Next application ready for online testing" should {
-    "return no application if htere is only one and it is a fast pass candidate" in{
+    "return no application if there is only one and it is a fast pass candidate" in{
```
Comment (henricook) [inline it/repositories/OnlineTestRepositorySpec.scala]: This is pretty controversial

File: app/repositories/package.scala [hunk 2]
```diff
@@ -195,7 +195,7 @@ package object repositories {
     val preferredName = personalDetailsRoot.getAs[String]("preferredName").get
 
     val assistanceDetailsRoot = doc.getAs[BSONDocument]("assistance-details").get
-    val guaranteedInterview = assistanceDetailsRoot.getAs[String]("guaranteedInterview").contains("Yes")
+    val guaranteedInterview = assistanceDetailsRoot getAs[Boolean]("guaranteedInterview") getOrElse false
```
Comment (henricook) [inline app/repositories/package.scala]: I mean, this clearly works, but we use . notation every where else, did you just fancy a change?

File: it/repositories/OnlineTestRepositorySpec.scala [hunk 3]
```diff
@@ -121,6 +121,30 @@ class OnlineTestRepositorySpec extends MongoRepositorySpec {
...
+    "be correctly read from mongo" in {
+       createApplicationWithAllFields("userId", "appId", "frameworkId", "SUBMITTED", needsAdjustment = false,
+        adjustmentsConfirmed = false, timeExtensionAdjustments = false, fastPassApplicable = false, isGis = true
+      )
+
+      val onlineTestApplication = onlineTestRepo.nextApplicationReadyForOnlineTesting.futureValue
+
+      onlineTestApplication.isDefined mustBe true
+
+      inside (onlineTestApplication.get) { case OnlineTestApplication(applicationId, applicationStatus, userId,
```
Comment (henricook) [inline it/repositories/OnlineTestRepositorySpec.scala]: :+1: :+1:

File: app/repositories/package.scala (hunk 2 above)
Comment (benjstephenson) [inline app/repositories/package.scala]: I'll change it back, infix notation is a bit dangerous.

Comment (henricook) [inline app/repositories/package.scala]: Cool. Good to merge after this

**Comment by henricook** [comment]
Just waiting for tests

**Pull request merged**